### ISSUE TITLE
filebot role - Permissions issue

### ISSUE THREAD

**Issue by zhdenny** [Cloudbox/Community#132]
**Update: I'm dumb.....Dev already fixed this**
https://github.com/Cloudbox/Community/commit/68f58e0bf4

Hey, found a problem with the filebot role.....and an easy fix for it
In its current state, it will set filebot up with permissions 1000:1000

This isn't right and you'll get permission issues if you try to do a rename

To fix, these environmental variables need to be added
USER_ID    =    1001
GROUP_ID    =    1002
Basically, filebot doesn't like to use the standard PUID/PGUID environmental variable standards.....it uses those custom ones

Aside from that, there also does not appear to be a simple way to password protect it for reverse proxy scenarios.

https://github.com/jlesage/docker-filebot

**Comment by desimaniac** [comment]
Fixed in 68f58e0
